src/js/utils/MetronomeUtil.js

```diff
@@ -1,70 +1,79 @@
 /**
- * Parse an item into a Job or JobTree - This method will create sub trees if
- * needed to insert the item at the correct location
- * (based on id/path matching).
- * @param {{
- *          id:string,
- *          items:array<({id:string, items:array}|*)>,
- * }} parent tree to add item to
- * @param {{id:string}} item job to add to parent
- * @param {[type]} jobsAlreadyAdded hash of id and jobs of jobs that
- * have already been added to the parent
+ * Get the parent id (e.g. foo) form the iten id (e.g. foo.bar) by matching
```

Comment:
Sorry for nitpicking spelling, but while you're addressing @nLight's comments, you could fix a couple mistakes:
`form` => `from`
`iten` => `item`